scm/content.go

```diff
@@ -16,10 +18,11 @@ type (
 	// ContentParams provide parameters for creating and
 	// updating repository content.
 	ContentParams struct {
-		Ref     string
-		Branch  string
-		Message string
-		Data    []byte
+		Branch    string
+		Message   string
+		Data      []byte
+		SHA       string
```

Comment:
for consistency please rename `SHA` to `Sha` since we use the latter throughout the codebase.